doc/faq.rst

```diff
@@ -100,12 +100,37 @@ See :ref:`adding_graphical_models`.
 Will you add GPU support?
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-No, or at least not in the near future. The main reason is that GPU support
-will introduce many software dependencies and introduce platform specific
-issues. scikit-learn is designed to be easy to install on a wide variety of
-platforms. Outside of neural networks, GPUs don't play a large role in machine
-learning today, and much larger gains in speed can often be achieved by a
-careful choice of algorithms.
+Adding GPU support by default would introduce heavy harware-specific software
+dependencies which make it both harder for the average user to install
```

Comment:
and reimplementation of algorithms? No dependency can make our trees run on GPU as explained below, right?

Reply:
Indeed, the sentence was updated based on Christian's suggestion to address this.